docs/product/session-replay/replay-details.mdx

```diff
@@ -51,3 +51,14 @@ Find a complete list of built-in fields and custom tags associated with a replay
 ## Retention
 
 Replays are retained for 90 days in sentry.io (the same retention period as other event types, such as Errors and Transactions). The retention period is not configurable.
+
+## Inbound Filtering
+
+Session replay supports some inbound filtering rules.  Those rules are:
+
+- IP Address
+- Releases
+- Request URL
+- User-Agent
+
+**Note**: Due to a quirk in how Replay emits successful outcomes versus filtered and dropped outcomes, you may observe a larger than expected increase in filtered outcomes. This is not an error. Filtered outcomes are emitted per **event** whereas successful outcomes are emitted per **replay**.  A replay being a collection of events means there is a natural imbalance in the number of outcomes per category.
```

Comment:
How about we call it segment instead of events? Would that be more accurate?

```suggestion
**Note**: Due to a quirk in how Replay emits successful outcomes versus filtered and dropped outcomes, you may observe a larger than expected increase in filtered outcomes. This is not an error. Filtered outcomes are emitted per **segment** whereas successful outcomes are emitted per **replay**.  A replay being a collection of segments means there is a natural imbalance in the number of outcomes per category.
```

Reply:
I didn't know if that was too much "jargon" or if they would understand.  I'm happy to change it if you think its better.

Reply:
"**Note**: Due to a quirk in how Replay emits successful outcomes versus filtered and dropped outcomes, you may observe a larger than expected increase in filtered outcomes." should we say 'on your Stats page' to contextualize this? That's where this info lives for the user (see Stats in left-nav)

Reply:
I think 'segments' is better IMO. Calling a replay a collection of events is pretty confusing since we already have a notion of 'replay events' and that's tied to our billing, and in that world, one replay is one event.

Reply:
Although I will say that it's hard for me to parse this note and the takeaway. @vivianyentran @lizokm perhaps we can reword and shorten?